### PR TITLE
WINUI-2489 Change circular, polar and funnel UG based on latest API breaking changes.

### PR DESCRIPTION
**Pull request by LavanyaGowtham2021**
### Feature description ###

Changed circular, polar, pyramid, and funnel chart API's based on latest breaking changes.

### Analysis and design ###

https://syncfusion.atlassian.net/browse/WINUI-2489

https://syncfusion.atlassian.net/browse/WINUI-2496

https://syncfusion.atlassian.net/browse/WINUI-2497

### Solution description ###

1. Changed circular, polar, pyramid, and funnel chart API's based on the latest breaking changes.
2. Prepared images for the legend checkbox.
3. Tested and updated existing code snippet.

### CODE REVIEW

File: winui/Circular-Charts/Appearance.md [hunk 1]
```diff
@@ -11,26 +11,9 @@ documentation: ug
...
-* BlueChrome
-* GreenChrome
-* Elite
-* LightCandy
-* SandyBeach
-
-### Applying Palette
-
-Each palette applies a set of predefined brushes to the series in a predefined order. [Metro](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartColorPalette.html#Syncfusion_UI_Xaml_Charts_ChartColorPalette_Metro) palette is the default palette.
+Default palette applies a set of predefined brushes to the series in a predefined order.
```
Comment (MuneeshKumarG) [inline winui/Circular-Charts/Appearance.md:17]: @naveenkumar-sanjeevirayan : Change the sentence.

File: winui/Polar-Chart/Axis-Labels.md [hunk 2]
```diff
@@ -213,7 +95,7 @@ The appearance of the axis labels can be customized by using the [LabelTemplate]
 
 {% highlight xaml %}
 
-<chart:SfPolarChart>
+<chart:SfPolarChart x:Name="chart">
```
Comment (MuneeshKumarG) [inline winui/Polar-Chart/Axis-Labels.md:99]: Don't set name for chart, use parent element for resource usage.

File: winui/Pyramid-Chart/Selection.md [hunk 3]
```diff
@@ -97,48 +97,6 @@ this.Content = chart;
...
-
-{% endhighlight %}
-
-{% endtabs %}
-
-![Changing cursor while selection support in WinUI Chart](Selection_images/WinUI_chart_cursor.png)
-
 ## Events
 
 The following selection events are available in [SfPyramidChart](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.SfPyramidChart.html).
```
Comment (MuneeshKumarG) [inline winui/Pyramid-Chart/Selection.md:102]: @naveenkumar-sanjeevirayan : Currently selection events moved to selection behavior. Change sentence in all charts.

File: winui/Circular-Charts/Appearance.md (hunk 1 above)
Comment (MuneeshKumarG) [reply winui/Circular-Charts/Appearance.md:17]: @naveenkumar-sanjeevirayan  : Refer below PR.
https://github.com/syncfusion-content/winui-docs/pull/274/files#diff-85d8087422682cdf5ec05e9a5388808041e311ae9dd4147a8b8a128299bfec87

File: winui/Circular-Charts/Selection.md [hunk 4]
```diff
@@ -69,9 +78,15 @@ Circular chart allows to select single or multiple segment\series using [Type](h
...
-chart.Behaviors.Add(selection);
+PieSeries series = new PieSeries();
+DataPointSelectionBehavior selection = new DataPointSelectionBehavior()
+{
+	SelectionBrush = new SolidColorBrush(Colors.BlueViolet),
+    Type = SelectionType.Multiple
+};
+
+series.SelectionBehavior = selection;
+chart.Series.Add(series);
```
Comment (MuneeshKumarG) [inline winui/Circular-Charts/Selection.md:89]: @naveenkumar-sanjeevirayan : Inselection feature -> SelectedIndex, SelectedIndexes property not explined. explain that too in all chart.

File: winui/Circular-Charts/Appearance.md (hunk 1 above)
Comment (naveenkumar-sanjeevirayan) [reply winui/Circular-Charts/Appearance.md:17]: content changes updated.

File: winui/Polar-Chart/Axis-Labels.md (hunk 2 above)
Comment (naveenkumar-sanjeevirayan) [reply winui/Polar-Chart/Axis-Labels.md]: template changes are moved to the parent grid.

File: winui/Pyramid-Chart/Selection.md [hunk 5]
```diff
@@ -55,7 +55,7 @@ this.Content = chart;
 
 ## Multi-selection
 
-Pyramid chart provides support to select multiple segments by using [Type](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartSelectionBehavior.html#Syncfusion_UI_Xaml_Charts_ChartSelectionBehavior_Type) property as [MultiPoint](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.SelectionType.html#Syncfusion_UI_Xaml_Charts_SelectionType_MultiPoint).  By default, the value of [Type](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartSelectionBehavior.html#Syncfusion_UI_Xaml_Charts_ChartSelectionBehavior_Type) is [Point](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.SelectionType.html#Syncfusion_UI_Xaml_Charts_SelectionType_Point) and it is used for single selection.
+Pyramid chart provides support to select multiple segments by using [Type]() property as [Multiple]().  By default, the value of [Type]() is [Single]() and it is used for single selection.
```
[the remark this answers is not in the document]
Comment (naveenkumar-sanjeevirayan) [reply winui/Pyramid-Chart/Selection.md:59]: Added API link for Type property.

File: winui/Pyramid-Chart/Selection.md (hunk 3 above)
Comment (naveenkumar-sanjeevirayan) [reply winui/Pyramid-Chart/Selection.md]: Updated changes in all charts.

File: winui/Circular-Charts/Selection.md [hunk 6]
```diff
@@ -81,29 +96,20 @@ chart.Behaviors.Add(selection);
...
-* [SelectedIndex](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartSelectionChangingEventArgs.html#Syncfusion_UI_Xaml_Charts_ChartSelectionChangingEventArgs_SelectedIndex) - Gets the selected data point index.
-* [PreviousSelectedIndex](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartSelectionChangingEventArgs.html#Syncfusion_UI_Xaml_Charts_ChartSelectionChangingEventArgs_PreviousSelectedIndex) - Gets the previous selected data point index.
-* [IsDataPointSelection](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartSelectionChangingEventArgs.html#Syncfusion_UI_Xaml_Charts_ChartSelectionChangingEventArgs_IsDataPointSelection) - Gets a value that indicates whether the selection is segment selection or series selection.
+* [CurrentIndex]() - Gets the selected data point index.
+* [PreviousIndex]() - Gets the previous selected data point index.
+
 * [Cancel](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartSelectionChangingEventArgs.html#Syncfusion_UI_Xaml_Charts_ChartSelectionChangingEventArgs_Cancel) - Gets or Sets a value that indicates whether the selection should be canceled.
```
[the remark this answers is not in the document]
Comment (naveenkumar-sanjeevirayan) [reply winui/Circular-Charts/Selection.md]: API link removed for cancel

**Review by SyncfusionKarthikeyan — APPROVED** [review]
This is fine

File: winui/Circular-Charts/Appearance.md [hunk 7]
```diff
@@ -56,62 +39,31 @@ chart.Series.Add(series);
...
-
-{% endtabs %}
-
-![Palette support in WinUI Chart](Appearance_images/winui_pie_chart_palette.png)
+## Custom PaletteBrushes
 
-## Custom Palette
-
-[SfCircularChart](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.SfCircularChart.html) provides support to define own brushes for chart with preferred order by using the [CustomBrushes](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartColorModel.html#Syncfusion_UI_Xaml_Charts_ChartColorModel_CustomBrushes) property of [ChartColorModel](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartColorModel.html) and [Palette](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartBase.html#Syncfusion_UI_Xaml_Charts_ChartBase_Palette) value as [Custom](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartColorPalette.html#Syncfusion_UI_Xaml_Charts_ChartColorPalette_Custom).
+[SfCircularChart](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.SfCircularChart.html) provides support to define own brushes for series with preferred order by using the [PaletteBrushes]() property, as shown in the following code example.
```
Comment (Chrispineagunja) [inline winui/Circular-Charts/Appearance.md:44]: The [SfCircularChart](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.SfCircularChart.html) provides support to define own brushes for series with preferred order by using the [PaletteBrushes]() property as shown in the following code example.

**Pull request merged**